build/build-viewer.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const {promisify} = require('util');
+const readFileAsync = promisify(fs.readFile);
```

Comment:
just curious, is there a particular reason we're using the async versions in scripts?

Reply:
> just curious, is there a particular reason we're using the async versions in scripts?

just a vague sense that we don't have to block on reading/writing, but just tried it (still on an SSD) with all sync and the files are so small/the build is so quick, it makes zero difference :) With `await` now they're roughly interchangeable, so it doesn't matter to me. Want to switch?

Reply:
Nah it's fine, I just tend to use the sync ones for scripts to not mess with the promisification and async in scripts when everything is pretty much serial anyway and was curious if there was another benefit